ext/dom/php_dom.stub.php

```diff
@@ -246,8 +537,106 @@ public function append(...$nodes): void {}
     public function prepend(...$nodes): void {}
 }
 
-class DOMDocument implements DOMParentNode
+class DOMDocument extends DOMNode implements DOMParentNode
 {
+    /**
+     * @dynamic
+     * @readonly
+     */
+    public ?DOMDocumentType $doctype;
+
+    /**
+     * @dynamic
+     * @readonly
+     */
+    public DOMImplementation $implementation;
+
+    /**
+     * @dynamic
+     * @readonly
+     */
+    public ?DOMElement $documentElement;
+
+    /**
+     * @dynamic
+     * @readonly
+     * @deprecated
```

Comment:
A few properties are marked as deprecated, and I think it's useful to do so in the stubs as well